### PR TITLE
Update devnet to 0.1.21

### PR DESCRIPTION
**Pull request by JorikSchellekens**

### CODE REVIEW

File: test/tokenbridge.test.ts [hunk 1]
```diff
@@ -49,8 +49,7 @@ describe('TokenBridge', async function() {
   let tokenBridgeL2: StarknetContract;
   let rewAaveTokenL2: StarknetContract;
   // L1
-  let MockStarknetMessaging: ContractFactory;
-  let mockStarknetMessaging: Contract;
+  let mockStarknetMessagingAddress: string;
```
Comment (achab) [inline test/tokenbridge.test.ts:52]: cool!

File: contracts/test/IStarknetMessaging.sol [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        external
+        returns (bytes32);
+
+    /**
+      Starts the cancellation of an L1 to L2 message.
+      A message can be canceled messageCancellationDelay() seconds after this function is called.
+      Note: This function may only be called for a message that is currently pending and the caller
+      must be the sender of the that message.
+    */
+    function startL1ToL2MessageCancellation(
```
Comment (achab) [inline contracts/test/IStarknetMessaging.sol:31]: nice import from starknet-dai-bridge!

**Review by achab — APPROVED** [review]
looks good to me!

File: contracts/test/IStarknetMessaging.sol (hunk 2 above)
Comment (JorikSchellekens) [reply contracts/test/IStarknetMessaging.sol:31]: Yeah, starknet is just extending their messaging system. Good things happening

**Pull request merged**